guidelines/index.html

```diff
@@ -91,15 +91,15 @@ <h4>New Features in WCAG 2.2</h4>
 					<p>WCAG 2.2 extends WCAG 2.1 by adding new success criteria, definitions to support them, and guidelines to organize the additions. This additive approach helps to make it clear that sites which conform to WCAG 2.2 also conform to WCAG 2.1. The Accessibility Guidelines Working Group recommends that sites adopt WCAG 2.2 as their new conformance target, even if formal obligations mention previous versions, to provide improved accessibility and to anticipate future policy changes.</p>
 					<p>The following Success Criteria are new in WCAG 2.2:</p>
 					<ul>
-                        <li><a href="#accessible-authentication">Accessible Authentication</a></li>
-                        <li><a href="#accessible-authentication-no-exception">Accessible Authentication (No Exception)</a></li>
-                        <li><a href="#consistent-help">Consistent Help</a></li>
-                        <li><a href="#dragging-movements">Dragging Movements</a></li>
-                        <li><a href="#focus-appearance">Focus Appearance</a></li>
-                        <li><a href="#focus-not-obscured-minimum">Focus Not Obscured</a></li>
-                        <li><a href="#focus-not-obscured-enhanced">Focus Not Obscured (Enhanced)</a></li>
-                        <li><a href="#redundant-entry">Redundant Entry</a></li>    
-                        <li><a href="#target-size-minimum">Target Size (Minimum)</a></li> 
+						<li>2.4.11 <a href="#focus-appearance">Focus Appearance</a> (AA)</li>
+						<li>2.4.12 <a href="#focus-not-obscured-minimum">Focus Not Obscured</a> (AA)</li>
+						<li>2.4.13 <a href="#focus-not-obscured-enhanced">Focus Not Obscured (Enhanced)</a> (AAA)</li>
+						<li>3.2.6 <a href="#consistent-help">Consistent Help</a> (A)</li>
+						<li>3.3.7 <a href="#accessible-authentication">Accessible Authentication</a> (AA)</li>
+						<li>3.3.8 <a href="#accessible-authentication-no-exception">Accessible Authentication (No Exception)</a> (AAA)</li>
+						<li>3.3.9 <a href="#redundant-entry">Redundant Entry</a> (A)</li>
+						<li>3.5.7 <a href="#dragging-movements">Dragging Movements</a> (AA)</li>
+						<li>3.5.8 <a href="#target-size-minimum">Target Size (Minimum)</a> (AA)</li>
```

Comment:
This is 2.5.8 so should move up the list

Reply:
Fixed the PR.